tests/test_legacy_external_class.py

```diff
@@ -0,0 +1,63 @@
+from nose.tools import assert_true, assert_list_equal, raises
+from numpy.testing import assert_almost_equal
+import datajoint as dj
+from . import schema_legacy_external as modu
+
+
+def test_heading():
+    heading = modu.Simple().heading
+    assert_true('item' in heading)
+    assert_true(heading['item'].is_external)
+
+
+def test_insert_and_fetch():
+    original_list = [1, 3, 8]
+    modu.Simple().insert1(dict(simple=1, item=original_list))
+    # test fetch
+    q = (modu.Simple() & {'simple': 1}).fetch('item')[0]
+    assert_list_equal(list(q), original_list)
+    # test fetch1 as a tuple
+    q = (modu.Simple() & {'simple': 1}).fetch1('item')
+    assert_list_equal(list(q), original_list)
+    # test fetch1 as a dict
+    q = (modu.Simple() & {'simple': 1}).fetch1()
+    assert_list_equal(list(q['item']), original_list)
+    # test without cache
+    previous_cache = dj.config['cache']
+    dj.config['cache'] = None
+    q = (modu.Simple() & {'simple': 1}).fetch1()
+    assert_list_equal(list(q['item']), original_list)
+    # test with cache
+    dj.config['cache'] = previous_cache
+    q = (modu.Simple() & {'simple': 1}).fetch1()
+    assert_list_equal(list(q['item']), original_list)
+
+
+def test_populate():
+    image = modu.Image()
```

Comment:
Can we remove these `()` throughout this and the next two methods?

Reply:
Yes, we could but, when using the same table multiple times, it's more efficient to instantiate once.

Reply:
This particular snippet is inside `test_legacy_external_class`, which tests old functionality, so it's a good place to test whether the old syntax still works.